modules/ROOT/pages/debugging-kernel-crashes.adoc

```diff
@@ -0,0 +1,34 @@
+= Enabling kdump to debug kernel crashes
+
+. First, install `kexec-tools` using `rpm-ostree`.
+[source, bash]
++
+----
+sudo rpm-ostree install kexec-tools
+----
+
+. Next, memory must be reserved for the crash kernel during booting of the first kernel. Kernel arguments can be provided like this:
++
+[source, bash]
+----
+sudo rpm-ostree kargs --append='crashkernel=256M'
+----
+
+. Next, make sure the path in which the vmcore will be saved is somewhere in the `/var` directory of the FCOS stateroot. It is also possible to write the dump over the network; for additional information, consult the mkdumprd man page and the comments in /etc/kdump.conf.
++
+[source, bash]
+----
+sed -i "s/^path.*/path \/sysroot\/ostree\/deploy\/fedora-coreos\/var\/crash/" /etc/kdump.conf
+----
+
+. Next, reboot your system.
++
+
+. Finally, enable and activate the kdump systemd service. 
++
+[source, bash]
+----
+sudo systemctl enable kdump.service --now
```

Comment:
This is OK but I think it'd be better to do it *before* the reboot (then you don't need `--now`, but it also doesn't hurt).

Reply:
Oh I remember why I put this here: if this were done earlier, `kexec-tools` wouldn't be layered on the system yet.